src/backend/access/zedstore/zedstoream_handler.c

```diff
@@ -184,7 +184,7 @@ zedstoream_insert_internal(Relation relation, TupleTableSlot *slot, CommandId ci
 		elog(ERROR, "slot's attribute count doesn't match relcache entry");
 
 	if (speculative_token == INVALID_SPECULATIVE_TOKEN)
-		tid = zsbt_tuplebuffer_allocate_tid(relation, xid, cid);
+		tid = zsbt_tuplebuffer_allocate_tids(relation, xid, cid, 1);
```

Comment:
What does `1` mean here?

Reply:
1 is the number of tids we want to allocate.